spec/unit/product-manager.spec.js

```diff
@@ -130,4 +130,180 @@ describe('ProductManager', () => {
       ])
     })
   })
+
+  describe('anonymizing product', () => {
+    const productMock = {
+      id: 'product-id',
+      version: 100,
+      productType: {
+        typeId: 'product-type',
+        id: 'product-type-id'
+      },
+      masterData: {
+        current: {
+          name: {
+            en: 'Product Name'
+          },
+          description: {
+            en: 'Product Description'
+          },
+          slug: {
+            en: 'product-slug',
+            de: 'product-slug-de'
+          },
+          masterVariant: {
+            id: 1,
+            sku: '2838301109'
+          },
+          variants: []
+        },
+        staged: {
+          name: {
+            en: 'Product Name'
+          },
+          description: {
+            en: 'Product Description'
+          },
+          slug: {
+            en: 'product-slug'
+          },
+          masterVariant: {
+            id: 1,
+            sku: '2838301109'
+          }
+        },
+        published: false,
+        hasStagedChanges: false
+      }
+    }
+
+    beforeEach(() => {
+      productService = new ProductManager(utils.logger, {})
+    })
+
+    it.only('should anonymize product', () => {
```

Comment:
You should remove `it.only`.

Reply:
Removed c63c7ae - we can also add an eslint plugin which will check that we are not commiting `.only` tests